docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,382 @@
+
+```js
+...
+  schema: {
+    type: 'object',
+    properties: {
+      street: {
+        type: 'string'
+      }
+    }
+  }
+...
+```
+
+Let's see what the form looks like now that we've added this question. The webpack server should have automatically refreshed the page with your form, so go back to your browser to take a look at what's there.
+
+This is what you should see:
+![Street field](../images/street-field.png)
+
+Hooray, a form field has appeared! But how did the library know to render a text input with a label of `street` based on what we added to the `schema`?
+
+The data that the `street` field accepts is `string` data. For the basic data types, an automatic determination is made by the library code on what type of HTML form element to render. `string` data renders a text input, a field with an `enum` property renders a select, and `boolean` data renders checkbox.
+
+The label is taken from the name of the field in the `formConfig`. `street` becomes "street" in our form. But what if we want to change the name that appears to the user? Let's do that next.
+
+### Step 7: Adding the `uiSchema` object.
+
+While the `schema` is the main description of the fields of our form, there are UI-specific things we may want to change about how those fields appear. Remember, the `schema` is mainly just a description of the type of `data` of that field, and not as concerned with how that field is displayed to the user.
+
+That's where `uiSchema` comes in. You can think of the `uiSchema` object as a mirror to the `schema` one. It usually has the same fields as the `schema` object, with different properties within each field to specify UI-specific characteristics of that field.
+
+The `uiSchema` object is added to our `page` at the same level as the `schema` object:
+
+```js
+firstPage: {
+  path: 'first-chapter/first-page',
+  title: 'First Page',
+  schema: {
+    type: 'object',
+    properties: {
+      ...
+    }
+  },
+  uiSchmea: {
+
+  }
+}
+```
+
+Let's return to the example of changing the label text of the `street` field. We can pass a property to the `uiSchema` under `street` called `ui:title` that lets us specify the exact label text we want displayed. This is how your `formConfig` would look:
+
+```js
+...
+  schema: {
+    type: 'object',
+    properties: {
+      street: {
+        type: 'string'
+      }
+    }
+  },
+  uiSchema: {
+    street: {
+      'ui:title': 'Street'
+    }
+  }
+...
+```
+(Note: most properties that are passed to `uiSchema` are prefixed with `ui:`, and therefore need to be passed as a string to `formConfig`.)
+
+Take another look at your form. You'll see that the label text has changed from "street" to "Street" because we overrode the default label text by adding `'ui:title': 'Street'` to our `formConfig`.
+
+Now that we understand the basics of adding fields to our `formConfig`, let's explore some different types of questions!
```

Comment:
Totally fair. I guess it's just a stylistic difference :) Happy to take these kinds of things out if it's annoying to users! I personally like it when reading tutorials like this, but that could definitely just be me.

Reply:
I think they're useful if you're watching a video, but because the next thing is on the next line of the same page, it's not necessary here. That's the only reason I'm less enthusiastic about them here.

Reply:
That's fair!